python/tests/test_cb.py

```diff
@@ -1,11 +1,18 @@
 import pandas as pd
 import sklearn as sk
 import numpy as np
+# import os
```

Comment:
nit: remove un-used import

Reply:
I'll remove the comments on this file once I'm done addressing most comments/feedback. I've been using them as a debug entrypoint.

Reply:
done